kernel/src/loader/mod.rs

```diff
@@ -229,51 +322,93 @@ mod tests {
         assert_eq!(val, b'\0');
     }
 
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    #[test]
     // Elf64 image that prints hello world on x86_64.
-    fn make_elf_bin() -> Vec<u8> {
-        let mut v = Vec::new();
-        v.extend_from_slice(include_bytes!("test_elf.bin"));
-        v
+    fn test_load_kernel() {
```

Comment:
This test function and the next one are identical save for the value of the `GuestAddress`. It looks like they can be merged by specifying different values via `#cfg[...]`.